test/parallel/test-repl-persistent-history.js

```diff
@@ -212,7 +206,10 @@ function cleanupTmpFile() {
 
 // Copy our fixture to the tmp directory
 fs.createReadStream(historyFixturePath)
-  .pipe(fs.createWriteStream(historyPath)).on('unpipe', () => runTest());
+  .pipe(fs.createWriteStream(historyPath)).on('unpipe', runTest);
+
+// redefine runTest function that uses common.mustCall (https://github.com/nodejs/node/tree/master/test#mustcallfn-expected)  
+runTest = common.mustCall(runTest, numtests);
```

Comment:
Could you please name the wrapped version something else and update the call points respectively?